db/migrate/20170221093606_update_old_redirect_unpublishings.rb

```diff
@@ -0,0 +1,24 @@
+class UpdateOldRedirectUnpublishings < ActiveRecord::Migration[5.0]
+  def up
+    Unpublishing
+      .where(type: :redirect)
+      .where(redirects: nil)
+      .find_each do |unpublishing|
+        # this magical line works because the 'redirects' getter is overriden
+        # to generate a JSON blob automatically if it is nil within the
+        # database
+        unpublishing.update_attribute(:redirects, unpublishing.redirects)
+      end
+  end
+
+  def down
```

Comment:
others might have (better, more informed) opinions, but i don't think a `down` migration is the place to have that safety net. @dougdroper @kevindew whatchu reckon?